src/components/AddOffering/AddItems/ItemImagesUpload.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import CloudUploadOutlinedIcon from "@material-ui/icons/CloudUploadOutlined";
+import IconButton from "@material-ui/core/IconButton";
+import ClearIcon from "@material-ui/icons/Clear";
+
+const useStyles = (theme) => ({
+  dropzone: {
+    border: `2px dashed ${theme.palette.text.disabled}`,
+    padding: theme.spacing(3),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    "& > *": {
+      margin: theme.spacing(1),
+    },
+  },
+  dropzoneDrop: {
+    backgroundColor: "#DBEAF8",
+    border: `2px dashed ${theme.palette.info.main}`,
+  },
+  uploadIcon: {
+    fontSize: "5rem",
+    color: theme.palette.text.disabled,
+  },
+  dropText: {
+    fontWeight: 300,
+    fontSize: "1.6rem",
+    color: theme.palette.text.secondary,
+  },
+  browseButton: {
+    color: theme.palette.info.main,
+    border: `1px solid ${theme.palette.info.main}`,
+    fontSize: "0.875rem",
+    lineHeight: "1.75",
+    borderRadius: "4px",
+    textTransform: "uppercase",
+    padding: "5px 15px",
+    boxSizing: "border-box",
+    cursor: "pointer",
+  },
+  input: {
+    display: "none",
+  },
+  imgPreview: {
+    height: 100,
+  },
+  imgContainer: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  deleteButton: {
+    margin: 0,
+    padding: 0,
+    position: "absolute",
+    right: "0",
+    top: "0",
+    color: "red",
+  },
+  imgDiv: {
+    margin: theme.spacing(1),
+    position: "relative",
+  },
+});
+
+class ItemImagesUpload extends React.Component {
```

Comment:
Code for this Component duplicates `Step2.js` in `/AddPosting` folder, but `useStyles` is tweaked somewhat